src/Tizen.NUI.Components/Controls/Button.Internal.cs

```diff
@@ -413,22 +424,12 @@ private void OnTextOrIconUpdated()
             }
 
             // If the button has fixed width and the text is not empty, the text should not exceed button boundary.
-            if (WidthSpecification >= 0 && !String.IsNullOrEmpty(buttonText.Text))
+            if (WidthSpecification != LayoutParamPolicies.WrapContent && !String.IsNullOrEmpty(buttonText.Text))
```

Comment:
Don't we need to check about 'MatchParent' property?

Reply:
In case `MatchParent`, the `size` parameter is equals to the parent size so it can calculate text's maximum size.